dir.c

```diff
@@ -935,17 +947,27 @@ dir_seek(VALUE dir, VALUE pos)
 
 #ifdef HAVE_SEEKDIR
 /*
- *  call-seq:
- *     dir.pos = integer  -> integer
+ * call-seq:
+ *   pos = position -> integer
+ *
+ * Sets the position in +self+ and returns +position+;
+ * the new position is:
+ *
+ * - The given integer +position+, if it is in +self+.
+ * - End-of-stream, if +position+ too large.
+ * - Zero, if +position+ is negative.
```

Comment:
We need to change this to talk about undefined behavior like `#seek`.

Reply:
Fixed.